rbc/src/test/java/com/library/rbc/controller/bookController/BookControllerShould.java

```diff
@@ -0,0 +1,34 @@
+package com.library.rbc.controller.bookController;
+
+import com.library.rbc.controller.BookController;
+import com.library.rbc.service.BookService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static com.library.rbc.controller.bookController.BookControllerSetUp.createBookDtos;
+import static com.library.rbc.controller.bookController.BookControllerSetUp.createResponseEntity;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class BookControllerShould {
+
+    @InjectMocks
+    private BookController bookController;
+
+    @Mock
+    private BookService bookService;
+
+    @Test
+    void getAllBooks() {
+        var bookDtos = createBookDtos();
```

Comment:
This is minor but consider making an agreement on project level on when to use var vs explicit type.

Reply:
We agreed to use var when working with long and complex types to simplify the code and improve readability.